src/vanilla/Model/MethodTS.cs

```diff
@@ -59,6 +59,45 @@ public override Parameter Add(Parameter item)
             return parameter;
         }
 
+        [JsonIgnore]
+        public string SchemeHostAndPort => (CodeModel as CodeModelTS).SchemeHostAndPort;
```

Comment:
If this expected to never fail I prefer a cast. Possibly a CodeModelTS property to contain that?

Reply:
Fair enough.